antenna/ext/s3/crashstorage.py

```diff
@@ -141,3 +143,33 @@ def save_crash(self, crash_report):
 
         # Save raw crash
         self.save_raw_crash(crash_id, raw_crash)
+
+    def load_crash(self, crash_id):
+        """Load crash data."""
+        raw_crash = {}
+        dumps = {}
+
+        raw_crash_key = self._get_raw_crash_path(crash_id)
+        try:
+            raw_crash = json.loads(self.connection.load_file(raw_crash_key))
+        except self.connection.KeyNotFound as exc:
+            raise CrashIDNotFound(f"{crash_id} not found") from exc
+
+        dump_names_path = self._get_dump_names_path(crash_id)
+        try:
+            dump_names = json.loads(self.connection.load_file(dump_names_path))
+        except self.connection.KeyNotFound:
+            pass
```

Comment:
Yes. It should be consistent across the crash storage classes. It's a weird situation that shouldn't happen and I did it differently three times. I'll fix that.

Reply:
The `dump_names` file is created by Antenna when saving the files from a crash report--it's not part of the crash report payload.

Crash storage should save the `dump_names` file, then the dump files, then the `raw_cash` file. For example, here it is in the S3 code:

https://github.com/mozilla-services/antenna/blob/f9eff555064b4c67ceebf67f603dffd296b2dd14/antenna/ext/s3/crashstorage.py#L133-L143

Because of that, I'm not sure what scenario could happen such that there exists a `raw_crash`, but no `dump_names` when running the tests.

Generally, it's bad form to have code silently fail--it's better to log something, raise an exception, emit a metric, etc.